app/views/directives/notifications/notification-drawer-wrapper.html

```diff
@@ -7,7 +7,7 @@
   heading-include="{{$ctrl.headingInclude}}"
   notification-body-include="{{$ctrl.notificationBodyInclude}}"
   notification-groups="$ctrl.notificationGroups"
-  notification-track-field="uid"
+  notification-track-field="trackByID"
```

Comment:
Don't you need to set `trackByID` here for events?

https://github.com/openshift/origin-web-console/blob/master/app/scripts/directives/notifications/notificationDrawerWrapper.js#L161

Reply:
Yup! looking at that now, I'll have an update in a second.

Reply:
I think mapping `event.metadata.uid` is fine.  not sure if it really needs the same format of ` _.uniqueId('notification-') + Date.now()`

Reply:
yeah has to be this way with `count` increments & things.